### DISCUSSION TITLE
Is it possible to add a table with the MFE composer editor?

### DISCUSSION THREAD

**Discussion by lucascanna** [mia-platform/community#520] [Q&A] [answered]
### What product are you having troubles with?

Console

### What Console version are you using?

v13

### Description

Hi all!

I am using the MFE composer to implement the backoffice of the Dev Portal. By default I have the following modal to update the elements of the marketplace:
<img width="1513" alt="Screenshot 2024-07-16 at 10 33 36" src="https://github.com/user-attachments/assets/8b7e9cf8-444c-43e6-a873-e06f0af8c39d">

It seems that with this editor it is not possible to add a table as rich text. Is there a way of doing it? Behind the scenes, is it HTML, MD or any other standard format? 

Thanks!

**Comment by clabene** [comment]
HI @Lucascanna! Tables are not supported unfortunately.
The component is based upon a react wrapper around [Quill (version 1)](https://v1.quilljs.com/). Underneath, an HTML representation of the text is used. If you'd like to check the exact shape of the data, you can do so by inspecting the body of the HTTP call resulting from submitting the form.

**Reply by lucascanna** [reply]
Thanks @clabene! Is there any other back-kit component we can use to add the table capability?

**Reply by clabene** [reply, marked as answer]
Depending on your use case, you might try to assign the target field in your data-schema with format `html-editor`.
Fields that specify such option are rendered through editors specifically designed to write an HTML-like strings - including therefore tags such as `<table>`, `<tbody>`, ...

This feature is available since back-kit version 1.4.17.

The resulting configuration of your field should look like this:
```
{
  dataSchema: {
    type: "object",
    properties: {
       ...
       tableField: {
         type: "string",
         format: "html-editor",
         ...
       }
    }
  }
}
```

You may use no code section of MFC configurator to edit the field format via the "Shared Properties" tab
